console/console-init/ui/src/modules/project/components/ProjectDatatoolbarToggleGroup/ProjectDataToolbarToggleGroup.tsx

```diff
@@ -163,13 +165,15 @@ const ProjectToolbarToggleGroup: React.FunctionComponent<IProjectToolbarToggleGr
       </ToolbarItem>
       <ToolbarItem spacer={{ md: "spacerNone" }} data-codemods="true">
         <ToolbarFilter
+          id="project-data-togglegrp-type-filter"
           chips={typeSelected ? [typeSelected] : []}
           deleteChip={onDelete}
           categoryName="Type"
         >
           {filterSelected && filterSelected.toLowerCase() === "type" && (
             <DropdownWithToggle
-              id={"al-filter-dropdown-type"}
+              id={"project-data-togglegrp-type-dropdowntoggle"}
```

Comment:
revisit id and toggleId for being consistent
id=`project-data-togglegrp-type-dropdown`
toggleId=`project-data-togglegrp-type-dropdowntoggle`